modules/core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/message/BulkStatus.java

```diff
@@ -212,6 +217,9 @@ public void merge(@NotNull BulkStatus update) {
         if (update.errorMessage != null && errorMessage == null) {
             errorMessage = update.errorMessage;
         }
+        if (update.getErrorCode() != 0) {
```

Comment:
Once the field will be nullable, we should have:
```suggestion
        if (update.errorCode != null && errorCode == null) {
```

Reply:
We should access the field directly like for `errorMessage`.